examples/build.gradle.kts

```
@@ -133,6 +133,6 @@ tasks.register("updateAddressbooks") {
     dependsOn(
         "updateAddressbooksMainnet",
         "updateAddressbooksPreviewnet",
-        "updateAddressbooksTestnet"
+        "updateAddressbooksTestnet",
```

Does this need a trailing comma? Previously there was not a trailing comma.
```suggestion
        "updateAddressbooksTestnet"
```

---

it's spotless rule.